src/Core/src/Platform/Android/PickerManager.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System.Collections.Generic;
+using Android.Text;
+using Android.Text.Style;
+using Android.Views;
+using Android.Widget;
+using AView = Android.Views.View;
+
+namespace Microsoft.Maui
+{
+	internal static class PickerManager
```

Comment:
Will something like this be used by TimePicker? Not sure how much overhead there is on this?